docs/deployment.md

```diff
@@ -0,0 +1,49 @@
+# How to deploy
+
+### Run public docker image
+```bash
+docker run -d -P --name cerenode cerebellumnetwork/turnkey-private-blockchain-network
```

Comment:
Link to Validator Rules. We should not have `turnkey-private-blockchain-network` node here.